### PR TITLE
Make voyager chart namespaced

### PR DESCRIPTION
**Pull request by tamalsaha**

### CODE REVIEW

**Comment by tamalsaha** [comment]
@unguiculus chart is namespaced

File: stable/voyager/templates/deployment.yaml [hunk 1]
```diff
@@ -1,33 +1,49 @@
...
+        scheduler.alpha.kubernetes.io/critical-pod: ''
+{{- end }}
     spec:
-      serviceAccountName: {{ if .Values.rbac.create }}{{ template "fullname" . }}{{ else }}"{{ .Values.rbac.serviceAccountName }}"{{ end }}
+      serviceAccountName: {{ if .Values.rbac.create }}{{ template "voyager.fullname" . }}{{ else }}"{{ .Values.rbac.serviceAccountName }}"{{ end }}
       containers:
       - name: operator
-        image: {{ .Values.image }}:{{ .Values.imageTag }}
+        image: {{ .Values.operator.image }}:{{ .Values.operator.imageTag }}
+        imagePullPolicy: {{ default "" .Values.imagePullPolicy | quote }}
```
Comment (unguiculus) [inline stable/voyager/templates/deployment.yaml]: Remove `default` function. It's unnecessary.

**Review by unguiculus — CHANGES_REQUESTED** [review]

**Comment by tamalsaha** [comment]
Addressed comments. PTAL

**Review by unguiculus — APPROVED** [review]

**Pull request merged**